types/bid.yaml

```
@@ -0,0 +1,22 @@
+BuilderBid:
+  type: object
+  description: "The `BuilderBid` object from the Builder API spec."
+  properties:
+    header:
+      $ref: "../beacon-apis/types/bellatrix/execution_payload.yaml#/Bellatrix/ExecutionPayloadHeader"
+      description: "`ExecutionPayloadHeader` for validator to sign."
+    value:
+      $ref: "../beacon-apis/types/primitive.yaml#/Uint64"
+      description: "Payment in wei that will be paid to the feeRecipient account."
+    pubkey:
+      $ref: "../beacon-apis/types/primitive.yaml#/Pubkey"
+      description: "BLS public key of builder."
+
+SignedBuilderBid:
+  type: object
+  description: "The `SignedBuilderBid` object from the Builder API spec."
+  properties:
+    message:
+      $ref: "#/BuilderBid"
+    signature:
+      $ref: "../beacon-apis/types/primitive.yaml#/Signature"
```

```suggestion
      $ref: "../beacon-apis/types/primitive.yaml#/Signature"
      description: "BLS signature of the relay"      
```

Clarify that this is the signed by the relay?

---

There isn't a concept of a "relay" in this API though, the only specified entities are the CL client and the external builder.